Mathlib/Data/Int/Order/Basic.lean

```diff
@@ -550,5 +578,38 @@ theorem toNat_sub_of_le {a b : ℤ} (h : b ≤ a) : (toNat (a - b) : ℤ) = a -
 
 end Int
 
+section bit0_bit1
+variable {R}
+set_option linter.deprecated false
+
+-- The next four lemmas allow us to replace multiplication by a numeral with a `zsmul` expression.
+-- They are used by the `noncomm_ring` tactic, to normalise expressions before passing to `abel`.
```

Comment:
I know you're just moving this, but no they don't in lean 4. Maybe just drop the comment?